sources/packages/api/src/route-controllers/user/user.controller.ts

```diff
@@ -17,13 +18,17 @@ export class UserController extends BaseController {
   }
 
   @Get("/check-user")
-  async checkUser(@UserToken() userToken: IUserToken): Promise<boolean> {
+  async checkUser(@UserToken() userToken: IUserToken): Promise<string> {
```

Comment:
IMHO it would be better to keep returning boolean and change 28 to return boolean true.